src/main/scala/com/microsoft/hyperspace/index/sources/delta/DeltaLakeRelation.scala

```diff
@@ -144,4 +145,97 @@ class DeltaLakeRelation(spark: SparkSession, override val plan: LogicalRelation)
       /* modificationTime */ modificationTime,
       path)
   }
+
+  /**
+   * Retrieve the history of index log version and delta table version from the given index.
+   *
+   * @param index Index to retrieve the version info.
+   * @return List of (index log version, delta table version) in ascending order.
+   */
+  private def deltaLakeVersionHistory(index: IndexLogEntry): Seq[(Int, Int)] = {
+    val versions =
+      index.derivedDataset.properties.properties
+        .getOrElse(DeltaLakeConstants.DELTA_VERSION_HISTORY_PROPERTY, "")
+
+    // The value is comma separated versions - <index log version>:<delta table version>.
+    // e.g. "1:2,3:5,5:9"
```

Comment:
```scala
// Versions are processed in a reverse order to keep the higher index log version in case different index
// log versions refer to the same delta lake version.
// For example, "1:1, 2:2, 3:2" will become Seq((1, 1), (3, 2)).
```
, if I understand this correctly?